src/main/java/org/selenim/pages/Screenshot.java

```diff
@@ -0,0 +1,31 @@
+package org.selenim.pages;
+
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import org.apache.commons.io.FileUtils;
+
+public class Screenshot {
+    private Date date;
+    private SimpleDateFormat formatDate;
+    private String nameScreen;
+    private File screen;
```

Comment:
I don't see any reason to make them as class variables as you are using them in the method. Please move them to method